etlas/Distribution/Client/Update.hs

```diff
@@ -63,14 +64,20 @@ update verbosity _ repoCtxt _ | null (repoContextRepos repoCtxt) = do
 update verbosity updateFlags repoCtxt firstTime = do
   let repos       = repoContextRepos repoCtxt
       remoteRepos = catMaybes (map maybeRepoRemote repos)
+      remoteRepoName' repo
+        | Just (URIAuth "" "github.com" "") == (uriAuthority uri) =
```

Comment:
Instead of pattern matching on the URI, you can use the `remoteRepoGitIndexed` field to check if it's a Git-backed index.